### PR TITLE
Asp dot net blog post, adding additional claims

### PR DESCRIPTION
**Pull request by mooreds**

### CODE REVIEW

File: site/_posts/2020-04-27-sharing-custom-oauth-claims-with-a-asp-netcore-app.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+```shell
+dotnet publish -r win-x64 && SampleApp__ClientSecret=H4... bin/Debug/netcoreapp3.1/win-x64/publish/SampleApp.exe
+```
+
+If you head to `http://localhost:5000` and click on the "Secure" page, you'll see the claims. If you were previously signed in, you may need to sign out and sign in again. The role is present in the JWT, as you can see below:
+
+{% include _image.liquid src="/assets/img/blogs/adding-more-claims-asp-net/aspnetextended-roles-only-roles-highlighted.png" alt="The secure page after a user has been associated with a role." class="img-fluid" figure=false %}
+
+Since the role and other claims are encoded in the JWT, which is stored in a cookie, whenever you change any user data in FusionAuth, you'll need log out and log in as that user to see the changes reflected in the ASP.NET Core application.
```
Comment (pgeoly) [inline site/_posts/2020-04-27-sharing-custom-oauth-claims-with-a-asp-netcore-app.md:41]:
```suggestion
you'll need to log out and log in 
```

File: site/_posts/2020-04-27-sharing-custom-oauth-claims-with-a-asp-netcore-app.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+## Adding custom claims
+
+You can add in custom claims based on user data, the time of day or anything else. In FusionAuth, you add this data to a JWT with a [lambda](https://fusionauth.io/docs/v1/tech/lambdas/jwt-populate). This is a JavaScript function which receives the JWT object before it is signed and can modify it. Remember way back when we [created the user](https://fusionauth.io/blog/2020/04/28/dot-net-command-line-client) and specified their favorite color as a command line option? Well, we're going to send that information down to the ASP.NET Core web application so that it can display that very valuable data.
```
Comment (pgeoly) [inline site/_posts/2020-04-27-sharing-custom-oauth-claims-with-a-asp-netcore-app.md:45]: Delete "in" before custom claims
Change "which" to "that" - This is a JavaScript function that receives the JWT 
```suggestion
You can add custom claims based on user data,
```
```suggestion
This is a JavaScript function that receives the JWT object
```

File: site/_posts/2020-04-27-sharing-custom-oauth-claims-with-a-asp-netcore-app.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+## Viewing custom claims 
+
+Now if you head to `http://localhost:5000`, you should see something like this:
+
+{% include _image.liquid src="/assets/img/blogs/adding-more-claims-asp-net/aspnetextended-favcolor-highlighted.png" alt="The secure page after a the lambda makes the favoriteColor claim available." class="img-fluid" figure=false %}
+
+Of course, this isn't much use. 
+
+In a real world application you'd do more than display these claims. You might customize a user's display based on their favorite color. A user's role could gate functionality or information.
```
Comment (pgeoly) [inline site/_posts/2020-04-27-sharing-custom-oauth-claims-with-a-asp-netcore-app.md:91]:
```suggestion
In a real-world application, you'd do more than display these claims. You might customize a user's display based on their favorite color. A user's role could gate functionality or information.
```

File: site/_posts/2020-04-27-sharing-custom-oauth-claims-with-a-asp-netcore-app.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+## Conclusion
+
+This post is the end of our journey integrating FusionAuth and ASP.NET Core. Offloading your user management concerns to a identity management solution such as FusionAuth frees you to focus on the features and business logic of your application.
```
Comment (pgeoly) [inline site/_posts/2020-04-27-sharing-custom-oauth-claims-with-a-asp-netcore-app.md:95]:
```suggestion
This post is the end of our journey integrating FusionAuth and ASP.NET Core. Offloading your user management concerns to an identity management solution such as FusionAuth frees you to focus on the features and business logic of your application.
```

**Comment by mooreds** [comment]
Closing this one because I'm going to open up a non forked PR.

**Pull request closed without merging**